Expose scheduling information on talk page.

lelutin opened the pull request:
When ppl click on a talk name, it would be nice to see some information
about when and where it is happening. The default wafer templates do not
expose this information.

hodgestar commented:
Looks good to me. Thanks!

One of the Travis builds failed to start, so I've restarted it.

lelutin commented:
about the translation blocks, I was wondering if it wouldn't be better to use a blocktrans and include the data inside the block. (e.g. start a blocktrans before `Duration:` and end the block only after `mins`, and similarly for other informations on the same page. this would make it possible to inverse text and data for languages that use a different direction.

do you think that change would make sense?

hodgestar commented:
@lelutin I think using `blocktrans` here makes a lot of sense.

lelutin commented:
ok this is hopefully my last commit for this PR. I've changed translation to use blocktrans on info lines above the talk description. I've also added `Type:` as translatable text.

hodgestar commented on wafer/schedule/models.py:
```diff
@@ -206,6 +206,11 @@ def get_duration(self):
+    def get_duration_minutes(self):
+        """Return the duration in total number of minutes."""
+        duration = self.get_duration()
+        return int(duration['hours']*60 + duration['minutes'])
```
I think PEP8 likes spaces around operators like (so ` * 60`).

hodgestar commented:
Left a minor style nitpick but the blocktrans version looks great. Tx for also updating the talk type part.

lelutin commented:
oops you're right about the missing space around the operator. I've pushed a fix for that

hodgestar commented:
:+1:

Thanks!
